cloudevents/schemas/document-schema.avsc

```diff
@@ -264,99 +338,132 @@
                                   }
                                 ]
                               },
-                              "name": "metadata",
+                              "name": "envelopemetadata",
                               "doc": "CloudEvents metadata constraints"
+                            },
+                            {
+                              "type": "GenericRecord",
+                              "name": "envelopeoptions",
+                              "doc": "Envelope metadata constraints"
                             }
                           ]
                         }
                       ],
                       "doc": "Message format identifier. This attribute MUST be the same as the 'format' attribute of the definition group, but is not automatically inherited because definitions may be cross-referenced across groups"
```

Comment:
Is this deliberately still "format"? (There are many other occurrences as well.) I'm assuming that only *some* `format` attributes have been renamed to `envelope`, but I haven't quite got my head round which yet.

Reply:
I just went thru and tried to use a phrase like "envelope format" to qualify it a bit more to avoid confusion when 'format' is actually a valid attribute near by. See if that help.